chapters/packages.tex

```diff
@@ -336,23 +336,24 @@ \section{Multilingual Descriptions}\label{multilingual-descriptions}
 \end{nonnormative}
 
 The files to support translation must be provided by the developer of the library.
-They must be stored in a subdirectory of the \filename{Resources} directory of the library with the name \filename{Language}.
+They must be stored in the resources directory \filename{modelica://<LibraryName>/Resources/Language/}.
 
 Two kind of files have to be provided:
 \begin{enumerate}
-\item Template file \filename{<LibraryName>.pot}.
+\item Template file \filename{<LibraryName>.pot}, as the resource \filename{modelica://<LibraryName>/Resources/Language/<LibraryName>.pot}.
 It contains all necessary information to translate all descriptions, but no translations.
 The pattern \filename{<LibraryName>} denotes the toplevel class name of the library.
-\item One file for each supported language with the name \filename{<LibraryName>.<language>.po}.
+\item One file for each supported language with the name \filename{<LibraryName>.<language>.po}, as the resource \filename{modelica://<LibraryName>/Resources/Language/<LibraryName>.<language>.po}.
 This file is a copy of the associated template file, but extended with the translations in the specified language.
 The pattern \filename{<language>} stands for the ISO 639-1 language code, e.g. \filename{de} or \filename{en}.
 \end{enumerate}
 
 The files consist of a header and a body.
+The detailed format of these files is described in \href{https://www.gnu.org/software/gettext/manual/gettext.pdf}{GNU gettext}.
```

Comment:
A normal citation would be better for those who still read the PDF.

Reply:
> Maybe a normal citation would be better? (Currently, we only use `\href` directly in _revisions.tex_.)

Could do, but I wanted to be as close as possible to the markdown of the proposed specification text.

Another possibility is to to use href more; I can understand that we want to show the complete hyper-link text for Modelica-links in order to make modelica.org more visible - but for external links I don't see a similar reason.

Reply:
> Could do, but I wanted to be as close as possible to the markdown of the proposed specification text.

Why?

Reply:
> Another possibility is to to use href more; I can understand that we want to show the complete hyper-link text for Modelica-links in order to make modelica.org more visible - but for external links I don't see a similar reason.

Of course, one reason to use normal citations is that having a bibliography adds value to the document by itself.  However, in order to provide that value, the bibliography should be complete in the sense that there aren't any additional references hiding in the document.

Reply:
> > Could do, but I wanted to be as close as possible to the markdown of the proposed specification text.
> 
> Why?

Because there was a proposed specification text in markdown, and I assumed people thought that it was ok - so the goal was to copy that into the normal specification while preserving everything.

Reply:
Note that one of the nice benefits of using a proper reference is that we get the back-references in the bibliography.

Reply:
> Because there was a proposed specification text in markdown, and I assumed people thought that it was ok - so the goal was to copy that into the normal specification while preserving everything.

That goal could work in some cases, but I don't find it realistic or helpful in gerneral.  The design documents are of most value before getting to the MCP stage of preparing actual specification changes.  After that point, I think we want maximum freedom of responding to reviewer feedback on the specification changes, while only making sure that the markdown design documents remain a correct description of the design going into the specification.

The design documents should also serve as a deeper background for future reference, and as such the style and scope of presentation could differ significantly from the specification changes.  Therefore, I'd generally not expect the formulations in the specification changes to be direct copies of formulations in the design documents.

Reply:
Obviously we can update it, but I wanted the starting point to be as close to it as possible.